subsys/net/l2/openthread/Kconfig

```diff
@@ -286,10 +286,4 @@ config OPENTHREAD_MAC_SOFTWARE_CSMA_BACKOFF_ENABLE
 	  Set y to enable software CSMA backoff. The option can be disabled if
 	  the radio has hardware support for this feature (IEEE802154_HW_CSMA).
 
-config OPENTHREAD_PLATFORM_USEC_TIMER_ENABLE
```

Comment:
Do you recall what was the purpose of this config? Is it just that the option was added but not actually implemented on the Zephyr side?

Reply:
Exactly, it was added here https://github.com/zephyrproject-rtos/zephyr/pull/29366, but never used.